### PR TITLE
feat: Allow tu customize 3 geolocation parameters

### PR DESCRIPTION
**Pull request by zatteo**
Submodule target : https://github.com/cozy/cozy-flagship-app/pull/980

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -34,10 +38,36 @@ import {
   sendLogFile,
   startTracking,
   stopTracking,
+  getTrackingConfig,
+  setTrackingConfig
 } from './cozy-flagship-app/src/app/domain/geolocation/tracking';
 
 const devMode = true;
 
+export const AccuracySelect = ({ value, onValueChange }) => {
```
Comment (paultranvan) [inline App.tsx]: nit: that is exported but only use in same file

File: App.tsx [hunk 2]
```diff
@@ -86,6 +116,64 @@ function GeolocationSwitch() {
...
+          onChangeText={setElasticityMultiplier}
+          value={elasticityMultiplier}
+          keyboardType="numeric"
+        />
+      </View>
+      <AccuracySelect value={desiredAccuracy} onValueChange={setDesiredAccuracy} />
+      <Button
+        title="Sauvegarder la configuration"
+        color="goldenrod"
+        onPress={() => saveTrackingConfig()}
```
Comment (paultranvan) [inline App.tsx:171]: I thought having a Save button was so 2010 :smile:

File: App.tsx [hunk 3]
```diff
@@ -86,6 +116,64 @@ function GeolocationSwitch() {
...
+
+      console.log(trackingConfig)
+
+      setDistanceFilter(trackingConfig.distanceFilter.toString())
+      setElasticityMultiplier(trackingConfig.elasticityMultiplier.toString())
+      setDesiredAccuracy(trackingConfig.desiredAccuracy.toString())
+
+    };
+    setInitialTrackingConfig();
+  }, []);
```
Comment (paultranvan) [inline App.tsx:136]: Won't this be called each time we set a new value (without even pressing the save button) ?

Comment (zatteo) [reply App.tsx:136]: No because there is an empty [] in the dependency array. So it will be called only when the component is mounted.

**Review by paultranvan — APPROVED** [review]

**Pull request merged**